src/main/java/org/apache/accumulo/access/AccessExpressionImpl.java

```diff
@@ -71,9 +71,9 @@ public Authorizations getAuthorizations() {
   }
 
   /**
-   * Creates a column visibility for a Mutation from a string already encoded in UTF-8 bytes.
+   * Creates an AccessExpression for a Mutation from a string already encoded in UTF-8 bytes.
```

Comment:
Mutation exists here too. Might want to search for it, Column Visibility, etc. in the project and fix all of them.